### ISSUE TITLE
how to Use DIST to train PanNuke datase

### ISSUE THREAD

**Issue by xiaoshuomin** [vqdang/hover_net#189]
hello, I want to produce the classification result on DIST,but use DIST requires 536*536 input in tensorflow repository, as the PanNuke dataset has been cropped into 256*256, so how can it be modified to run train? The same problems include unet, segnet, micronet and so on. In addition, in the tensorflow branch, why the input size of different models is different, is this not rigorous?Thank you

**Comment by simongraham** [comment]
For the likes of DIST and U-Net, valid convolution is used, which means that the output size is smaller than the input size. For these networks, we worked from the bottom to the top (lowest spatial resolution to highest spatial resolution) and tried to match the input dimensions as close as possible. 

In the tensorflow repository, the input dimensions is twice as large as the input to the network. This is because we augment and then perform a random central crop. You can instead just use the original patch and perform reflective padding [here](https://github.com/vqdang/hover_net/blob/master/dataloader/train_loader.py#L120) by using mode='reflect'. You may ensure that the input and output dimensions are 256x256 by using padded convolution rather than valid convolution.
